### PR TITLE
Navigation drawer from app bar

### PR DESCRIPTION
**Pull request by PappBeata**
We have put the pieces (toolbar and drawer parts) in place. Now our structure follows the structure of an android studio standrard activity with navigation bar.

### CODE REVIEW

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/FirstFragment.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+ * Created by mozgaanna on 25/01/17.
+ */
+
+public class FirstFragment extends Fragment {
+
+    View contentView;
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        contentView = inflater.inflate(R.layout.first_layout, container, false);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/FirstFragment.java]: @PappBeata dont need to store the contentView as class variable, have it declared in the method

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/SecondFragment.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * Created by mozgaanna on 25/01/17.
+ */
+
+public class SecondFragment extends Fragment {
+
+    View contentView;
+    @Nullable
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        contentView = inflater.inflate(R.layout.second_layout, container, false);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/SecondFragment.java]: @PappBeata same in english

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/FirstFragment.java (hunk 1 above)
Comment (PappBeata) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/FirstFragment.java]: Done

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/SecondFragment.java (hunk 2 above)
Comment (PappBeata) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/fragment/SecondFragment.java]: Done

**Review by adamgyulavari — APPROVED** [review]

**Comment by adamgyulavari** [comment]
@PappBeata resolve conflicts :)

**Comment by PappBeata** [comment]
Should be ok. Sorry, I haven't press on commit after marking as solved...

File: app/src/main/res/drawable-v21/ic_menu_manage.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillColor="#FF000000"
+        android:pathData="M22.7,19l-9.1,-9.1c0.9,-2.3 0.4,-5 -1.5,-6.9 -2,-2 -5,-2.4 -7.4,-1.3L9,6 6,9 1.6,4.7C0.4,7.1 0.9,10.1 2.9,12.1c1.9,1.9 4.6,2.4 6.9,1.5l9.1,9.1c0.4,0.4 1,0.4 1.4,0l2.3,-2.3c0.5,-0.4 0.5,-1.1 0.1,-1.4z" />
+</vector>
```
Comment (adamgyulavari) [inline app/src/main/res/drawable-v21/ic_menu_manage.xml]: new line @PappBeata

File: app/src/main/res/layout/first_layout.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+    <TextView
+        android:text="You are on the first fragment layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="151dp"
+        android:id="@+id/firstLayout" />
+</RelativeLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/first_layout.xml]: new line @PappBeata

File: app/src/main/res/layout/second_layout.xml [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+    <TextView
+        android:text="You are on the second fragment layout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentTop="true"
+        android:layout_centerHorizontal="true"
+        android:layout_marginTop="151dp"
+        android:id="@+id/secondLayout" />
+</RelativeLayout>
```
Comment (adamgyulavari) [inline app/src/main/res/layout/second_layout.xml]: new line @PappBeata

File: app/src/main/res/drawable/side_nav_bar.xml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+    <gradient
+        android:angle="135"
+        android:centerColor="#4CAF50"
+        android:endColor="#2E7D32"
+        android:startColor="#81C784"
+        android:type="linear" />
+</shape>
```
Comment (adamgyulavari) [inline app/src/main/res/drawable/side_nav_bar.xml]: new line @PappBeata

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Comment by PappBeata** [comment]
Dependency change of master left unchanged. (-Lombok +android.support.design)

File: app/src/main/res/drawable/side_nav_bar.xml (hunk 6 above)
Comment (PappBeata) [reply app/src/main/res/drawable/side_nav_bar.xml]: New lines added.

File: app/src/main/res/layout/second_layout.xml (hunk 5 above)
Comment (PappBeata) [reply app/src/main/res/layout/second_layout.xml]: Addded

File: app/src/main/res/layout/first_layout.xml (hunk 4 above)
Comment (PappBeata) [reply app/src/main/res/layout/first_layout.xml]: Added

File: app/src/main/res/drawable-v21/ic_menu_manage.xml (hunk 3 above)
Comment (PappBeata) [reply app/src/main/res/drawable-v21/ic_menu_manage.xml]: Added.

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**